e2e/integration/usergrouptests.js

```diff
@@ -23,13 +26,28 @@ context( 'Test for UserGroup', () => {
         } )
     })
 
-    it( 'Test for search user group by name', () => {
+    it( 'Test for search user group by rover group', () => {
         cy.get( '#searchText' ).type( 'dsal-admin' )
         cy.get( 'tbody' ).within( () => {
             cy.get( '[data-label^="LDAP"]' ).first().should( 'have.text','dsal-admin')
```

Comment:
@mudit94 why is there dsal-admin mentioned here?

Reply:
basically it is trying to search that group. in the list

Reply:
@hybridx

Reply:
@mudit94 can you change it to something else please?